extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/DevServicesConfig.java

```diff
@@ -89,6 +89,20 @@ public class DevServicesConfig {
     @ConfigItem
     public Optional<String> javaOpts;
 
+    /**
+     * Show Keycloak log messages with a "Keycloak:" prefix.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean showLogs;
+
+    /**
+     * Keycloak start command.
+     * Use this property to experiment with Keycloak start options, see {@link https://www.keycloak.org/server/all-config}.
+     * Note it will be ignored when loading legacy Keycloak WildFly images.
+     */
+    @ConfigItem
+    public Optional<String> startCommand;
```

Comment:
Default it to what's in `KEYCLOAK_QUARKUS_START_CMD` so docs say what's used?

Reply:
Hi @gastaldi I was thinking about it but I hesitated because I'm not sure yet how to combine it with the optional adding of another option at https://github.com/quarkusio/quarkus/pull/28101/files#diff-67404a43520250b1ebd5a0c945e070e60d70d4a3ba06b15e3fac747f820320ccR446

but also I was not sure the time for adding `start --storage=chm --http-enabled=true --hostname-strict=false --hostname-strict-https=false` into JavaDocs was right :-), may be later when this sequence become more compact and quite settled ?